### PR TITLE
Merge mandos pr

### PR DESCRIPTION
**Pull request by dylanratcliffe**
This is a re-do of #112

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,8 +1,7 @@
 source 'https://rubygems.org'
 
-gem 'rubygems-tasks'
+gemspec
+
 gem 'git'
```
Comment (Mandos) [inline Gemfile]: I think 5,6 and 7 line should be only in gemspec.

File: Gemfile [hunk 2]
```diff
@@ -1,8 +1,7 @@
...
 gem 'rake'
 gem 'rubocop', "~> 0.39.0" # in line with puppet gem and its rubocop.yml we're using
```
Comment (Mandos) [inline Gemfile]: Shouldn't it be moved to gem spec file, same with git and rake gems.

Comment (dylanratcliffe) [reply Gemfile]: Yep, good call, fixed

**Comment by dylanratcliffe** [comment]
@Mandos How about now?

**Comment by Mandos** [comment]
Looks good, thanks.

**Pull request merged**